Controller/AuthorizeController.php

```diff
@@ -50,44 +114,85 @@ public function setContainer(ContainerInterface $container = null)
         $this->container = $container;
```

Comment:
Put the `setContainer()` method back.

Not sure if I went far enough though. Now `setContainer()` sets the container, but I didn't revert the dependency injection changes. So the container can be set, but it's never used.

Reply:
Before these changes, things like the request, session, `AuthorizeFormHandler`, etc. were retrieved from the container as needed. Now those things are injected into the constructor method and so the container isn't actually used in this class.